src/components/ArticlesList.jsx

```diff
@@ -1,6 +1,22 @@
+import { useEffect, useState } from "react";
 import { Button } from "react-bootstrap";
+import { fetchArticles } from "../utils/Api";
+import ArticleCard from "./ArticleCard";
 
 export default function ArticlesList() {
+  const [articles, setArticles] = useState([]);
+
+  const searchForArticles = () => {
+    fetchArticles().then((response) => {
+      
+      setArticles(response.data.article);
+    });
+  };
+
+  useEffect(() => {
+    searchForArticles();
```

Comment:
While this API call is being made, it would be good to indicate to the user that the page is loading
You should use an `isLoading` state to conditionally render a loading component
Here's some notes on loading patterns: https://notes.northcoders.com/courses/js-front-end/useEffect#loading-patterns

Reply:
Superb, thanks for the link :D 
Sorted and enjoyed it :)